src/main/java/com/google/jenkins/plugins/k8sengine/KubernetesEngineBuilder.java

```diff
@@ -200,6 +216,40 @@ public void perform(
     }
   }
 
+  /**
+   * Verify the application of the supplied manifest objects and return true if it succeeded.
+   *
+   * @param kubectl the KubectlWrapper for running the queries on the kubernetes cluster.
```

Comment:
Please make sure to use capitalization consistent with the javadoc throughout the rest of the code.